src/App.js

```diff
@@ -46,9 +71,11 @@ const App = ({ initialPath }) => {
         />
       </Helmet>
       <Analytics>
-        <Grommet theme={THEMES[themeName || 'grommet']}>
-          <Content />
-        </Grommet>
+        <ThemeSwitchContext.Provider value={themeSwitchContextValue}>
+          <Grommet theme={THEMES[themeName || 'grommet']}>
```

Comment:
```suggestion
          <Grommet theme={THEMES[themeName]}>
```
If themeName always defaults to grommet, we probably don't need this fallback anymore.